tests/manager/manager-var-dump-001.phpt

```diff
@@ -45,10 +45,10 @@ object(MongoDB\Driver\Manager)#%d (%d) {
       ["is_passive"]=>
       bool(false)
       ["tags"]=>
-      array(0) {
+      object(stdClass)#%d (0) {
```

Comment:
@bjori: One more question. If `Server::getTags()` and `Server::getInfo()` return the tags and isMaster response, respectively, shouldn't those return stdClass instances by default? Or should they be consistent with our debug handler and decode BSON objects as arrays? This would primarily affect the method signature for `Server::getTags()`, since it's currently documented as returning an array.

I'm leaning towards using arrays for both of this, since they're both largely used for diagnostics and shouldn't have any array/document confusion.